pom.xml

```diff
@@ -40,13 +40,42 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.2.0</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+		</dependency>
+
+		<dependency>
```

Comment:
удалите вот эту зависимость !
и не будет этой ошибки
Выдает ошибку "Liquibase failed to start because no changelog could be found at 'classpath:/db/changelog/db.changelog-master.yaml'." Не могу решить, помогите разобраться